e2e/tests/dump-restore-test.rs

```diff
@@ -32,15 +33,19 @@ fn psql_file(container: &PostgresContainer, db: &str, username: &str, path: &Pat
         .arg("--echo-all")
         .args(["-v", "ON_ERROR_STOP=1"])
         .args(["-f", path.to_str().unwrap()])
-        .spawn()
-        .unwrap()
-        .wait()
+        .output()
         .unwrap();
+    String::from_utf8(output.stdout)
```

Comment:
could be useful to output stderr as well

Reply:
Ya, the problem that I ran into is that _ideally_ we would pipe stderr to stdout and output both like that. If we output them separately, then the output will not be interleaved. Unfortunately std doesn't provide tools to do this, but it looks like there might be some crates to handle that...

Reply:
done